src/scripts/dungeons/Dungeon.ts

```diff
@@ -137,10 +137,14 @@ class Dungeon {
             if (loot.requirement && !loot.requirement.isCompleted()) {
                 return 0;
             }
-            // Minimum of 1 times cleared for division
+            // Minimum of 1 times cleared for division, maximum of 500 times cleared
             const timesCleared = Math.min(500, Math.max(1, App.game.statistics.dungeonsCleared[GameConstants.getDungeonIndex(this.name)]()));
             // Calculate total weight based on times cleared, minimum weight being original number specified
-            return Math.max(loot.weight, Math.pow(10, loot.weight) / timesCleared) + 1 || 1;
+            if (App.game.challenges.list.hardDungeons.active()) {
+                return Math.max(loot.weight, Math.pow(10, Math.pow(loot.weight, loot.weight)) / timesCleared) + 1 || 1;
```

Comment:
I think this might be a little too intense? 😅 

![image](https://user-images.githubusercontent.com/7288322/147204618-a573350d-d52e-44b0-8a75-b59e6ae449c4.png)

Reply:
😆 Yes, it was inteded to be OP, no more whining from Challenge Mode players. I figured you might want it changed, but thought leaving it high to start was ok.